spec/requests/solidus_pay_tomorrow/api/webhooks_controller_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusPayTomorrow::Api::WebhooksController, type: :request do
+  subject(:notify_pending_call) do
+    post "/pay_tomorrow/notify", params: pending_params
+  end
+
+  let(:payment_method) { create(:pt_payment_method) }
+  let(:payment_source) { create(:pt_payment_source, payment_method: payment_method) }
+
+  describe 'POST /pay_tomorrow/notify' do
+    context 'when uuid verification fails' do
+      let(:pending_params) {
+        { txn_type: 'cart',
+          payment_status: 'pending',
+          pt_currency: 'USD',
+          pt_amount: 100,
+          uuid: 'an invalid uuid' }
+      }
+
+      it 'gracefully returns with error message' do
+        notify_pending_call
+        expect(response.status).to be(422)
+        expect(JSON(response.body)['message']).to \
+          eq("Couldn't find order with uuid: an invalid uuid")
+      end
```

Comment:
Usually, webhooks continue to hit the endpoint while it's returning an error.
Can you verify if this will happen even for pay_tomorrow? If so, we should return 200(or 404) to avoid it.

Reply:
I have not noticed in PT yet. I'll double check though

Reply:
Yeah right, it's retrying with 404 and 422
Fine with 200 so I'll have to make it 200

Reply:
![retry_on_422](https://user-images.githubusercontent.com/9041163/209944358-2539d602-8229-4726-a166-3ce0cbb24176.png)
![retry_with_200](https://user-images.githubusercontent.com/9041163/209944370-2626215f-c59c-49ae-a9db-80b5e5d46bcc.png)
![retry_with_404](https://user-images.githubusercontent.com/9041163/209944378-931d3910-ea72-41df-bc80-b370037bef1b.png)

Reply:
Fixed in https://github.com/solidusio-contrib/solidus_pay_tomorrow/pull/26/commits/9e68cf85ab013c161a0dec390b3520f5ea34b2bd